docs/pages/usage-billing.mdx

```diff
@@ -49,6 +49,11 @@ measurement.
 The code that aggregates and anonymizes this data can be found in our [GitHub
 repository](https://github.com/gravitational/teleport/tree/master/lib/usagereporter/teleport/aggregating).
 
+For Teleport Auth services that run in a restricted network environment you
+can set the `TELEPORT_REPORTING_HTTPS_PROXY` and `TELEPORT_REPORTING_HTTP_PROXY`
```

Comment:
I would say what these two environment variables do before mentioning them. For example, we could say something like:

```
you can configure the Auth Service instances to send usage data through a proxy in your network. To do so, assign the variables...
```

That way, we can be more explicit about the behavior that the variables configure. I wouldn't define these variables only in terms of other variables, since the user would need to look up the way those _other_ variables work by searching through the docs.

Reply:
@ptgott please see update